main.go

```diff
@@ -40,17 +40,19 @@ running 'git config cockroach.remote REMOTE-NAME'.
 
 Options:
 
-      --continue           resume an in-progress backport
-      --abort              cancel an in-progress backport
-  -c, --commit <commit>    only cherry-pick the mentioned commits
-  -r, --release <release>  select release to backport to
-  -f, --force              live on the edge
-      --help               display this help
+       --continue           resume an in-progress backport
+       --abort              cancel an in-progress backport
+  -c,  --commit <commit>    only cherry-pick the mentioned commits
+  -r,  --release <release>  select release to backport to
+  -br, --branch <branch>    select the branch to backport to
```

Comment:
`-br` may not work as expected. Usually only one letter is used for short options. Also in the code below you used `"s"`. Maybe use `-b` instead?